sui_core/tests/staged/sui.yaml

```diff
@@ -456,32 +456,37 @@ SuiError:
           - object_id:
               TYPENAME: ObjectID
     4:
-      UnexpectedOwnerType: UNIT
+      MovePackageAsObject:
```

Comment:
A lot of these are due to reordering of the error types,
- this is bound to create a ton of churn as new enum options introduced early in the giant `SuiError` enum invalidate everything,
- we do generate a single error type for everything.

Here's [an example](https://twitter.com/GergelyOrosz/status/1502947315279187979) of consequence this churn can have.
=> do we want to open an issue to start breaking up the `SuiError` type in nested types have those grouped by area where they can be triggered?